### PR TITLE
Added cypress tests for sidebar component in homepage

### LINKED ISSUE #324: Add cypress tests for sidebar component inside homepage

### PR DESCRIPTION
**Pull request by Janvi-Thakkar**
## Description
Added cypress tests for drawer side and static sidebar as per screen size

## Related Issue
Fixes #324 

## Screenshots or GIF (In case of UI changes):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Niweera** [comment]
@Janvi-Thakkar can you add a gif (using recordit). You only added a png which does not give us much info.

**Comment by Janvi-Thakkar** [comment]
> @Janvi-Thakkar can you add a gif (using recordit). You only added a png which does not give us much info.

Recordit is not working in my laptop. so I tried with different software.
**Also, the screens in which 404 is showing is right now because we don't have those routes added in the routes folders. once we would be done with the development. we would have those routes in place.**

https://user-images.githubusercontent.com/55620053/177035219-06649bda-5e42-4160-a39c-adc8ca39ebef.mp4

File: src/components/HomePage/index.js [hunk 1]
```diff
@@ -86,29 +95,34 @@ function HomePage({ background = "white", textColor = "black" }) {
...
-              >
+            </List>
+          </Grid>
+        </Grid>
 
-                  <Grid item className={classes.outerSideBar}>
-                      <SideBar open={openMenu} toggleSlider={toggleSlider} />
-                  </Grid>
-              </Grid>}
+        {window.innerWidth > 750 && (
```
Comment (Niweera) [inline src/components/HomePage/index.js:102]: can you explain what is the purpose of using `window.innerWidth` instead using [material UI breakpoints](https://mui.com/material-ui/customization/breakpoints/)?

Comment (Janvi-Thakkar) [reply src/components/HomePage/index.js:102]: for conditional rendering using a mui breakpoint of 750px, I had to declare it in the CSS style of the class as there is no breakpoint provided for 750px by mui. On the other hand, the window.innerWidth was allowing me to give conditions directly. There is no other purpose behind doing it. If window.innerWidth does not seem feasible to you please let me know. I would change according to your suggestion.

Comment (Janvi-Thakkar) [reply src/components/HomePage/index.js:102]: @Niweera Thank you, I will do!

Comment (Niweera) [reply src/components/HomePage/index.js:102]: cool

**Review by Niweera — DISMISSED** [review]
can you resolve the merge conflict?

**Comment by Janvi-Thakkar** [comment]
> can you resolve the merge conflict?

Done

**Review by Niweera — APPROVED** [review]

**Pull request merged**